src/js/add_input_icon.js

```diff
@@ -240,14 +238,31 @@ async function addRelayIconToInput(emailInput) {
       sendInPageEvent("click", "input-menu-manage-all-aliases-btn");
     });
 
+    //Create "Get unlimited aliases" link
+    const { fxaSubscriptionsUrl } = await browser.storage.local.get("fxaSubscriptionsUrl");
+    const { premiumProdId } = await browser.storage.local.get("premiumProdId");
+    const { premiumPriceId } = await browser.storage.local.get("premiumPriceId");
```

Comment:
`browser.storage.local.get` will let you pass an array of strings so you can make just 1 call: `browser.storage.local.get(["fxaSubscriptionsUrl", "premiumProdId", "premiumPriceId"]);`

Looks like that will return a results object containing all 3 objects inside it.

Reply:
Should I do `const { fxaSubscriptionsUrl, premiumProdId, premiumPriceId } = await browser.storage.local.get("fxaSubscriptionsUrl", "premiumProdId", "premiumPriceId");
` ?